docs/bugs_and_glitches.md

```diff
@@ -51,7 +51,7 @@ Fixes in the [multi-player battle engine](#multi-player-battle-engine) category
   - [Glacier Badge may not boost Special Defense depending on the value of Special Attack](#glacier-badge-may-not-boost-special-defense-depending-on-the-value-of-special-attack)
   - ["Smart" AI encourages Mean Look if its own Pokémon is badly poisoned](#smart-ai-encourages-mean-look-if-its-own-pokémon-is-badly-poisoned)
   - ["Smart" AI discourages Conversion2 after the first turn](#smart-ai-discourages-conversion2-after-the-first-turn)
-  - ["Smart" AI does not encourage Solar Beam, Flame Wheel, or Moonlight during Sunny Day](#smart-ai-does-not-encourage-solar-beam-flame-wheel-or-moonlight-during-sunny-day)
+  - ["Smart" AI does not encourage Solar Beam, Flame Wheel, and Moonlight during Sunny Day](#smart-ai-does-not-encourage-solar-beam-flame-wheel-and-moonlight-during-sunny-day)
```

Comment:
"or" is grammatical here; logically you'd use "nor" but that's archaic.

Reply:
This is simply fixing a link to an existing section, I'd consider fixing this "properly" in a different commit/PR.

Reply:
This is simply fixing a link to an existing section, I'd consider fixing this "properly" in a different commit/PR.

Reply:
I mean, just put it back to "or". Not suggesting to actually use "nor".

Reply:
I can still change it here since it was just made to fix a broken link.